x-pack/plugins/apm/public/components/app/ServiceDetails/ServiceIntegrations/MachineLearningFlyout/index.tsx

```diff
@@ -52,10 +52,14 @@ export class MachineLearningFlyout extends Component<Props, State> {
     }, 0);
   }
 
-  public onClickCreate = async () => {
+  public onClickCreate = async ({
+    transactionType
+  }: {
+    transactionType: string;
+  }) => {
     this.setState({ isCreatingJob: true });
     try {
-      const { serviceName, transactionType } = this.props.urlParams;
+      const { serviceName } = this.props.urlParams;
       if (!serviceName || !transactionType) {
```

Comment:
is this check still relevant if it's not coming from the urlParams?

Reply:
No, I don't think so. Thanks, good catch 👍